Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -9599,6 +9600,7 @@
             <LayoutButtons Face="AP_BirthRaptor" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train3" Row="0" Column="0"/>
             <LayoutButtons Face="AP_BirthSwarmling" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train2" Row="0" Column="0"/>
             <LayoutButtons Face="AP_BirthZergling" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train1" Row="0" Column="0"/>
+            <LayoutButtons Face="AP_BirthDrone" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train4" Row="2" Column="1"/>
             <LayoutButtons Face="AP_BirthCorpser" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train6" Row="0" Column="1"/>
             <LayoutButtons Face="AP_BirthVile" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train7" Row="0" Column="1"/>
             <LayoutButtons Face="AP_BirthRoach" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train5" Row="0" Column="1"/>

```

Comment:
I'd put Drone into first row before lings. Cancel Last should be at bottom-right slot

Reply:
https://github.com/user-attachments/assets/cbaae1b1-acf1-455e-9505-5ed8a2fbc93e

Made requested changes to to command card, and fixed rallying bug.
(Also made a slight nerf to resource efficiency in most recent commit)